zmon-controller-app/src/main/java/org/zalando/zmon/service/impl/ZMonServiceImpl.java

```diff
@@ -177,7 +180,28 @@ private ExecutionStatus buildStatus(int alertsActive, final Map<String, Response
     @Override
     public List<CheckDefinition> getCheckDefinitions(final DefinitionStatus status,
                                                      final List<Integer> checkDefinitionIds) {
-        return checkDefinitionSProc.getCheckDefinitions(status, checkDefinitionIds);
+        List<CheckDefinition> checkDefinitions = checkDefinitionSProc.getCheckDefinitions(status, checkDefinitionIds);
+        return enrichCheckDefinitionsWithCriticality(checkDefinitions);
+    }
+
+    private List<CheckDefinition> enrichCheckDefinitionsWithCriticality(List<CheckDefinition> checkDefinitions) {
```

Comment:
private methods usually go after public and protected ones

Reply:
I'm also up for it. But I tried to follow the approach that is present in the current file. And what I saw, is that private methods following the public one, where they are used.
What do you think about rearranging all methods in the class to have it in one proper style?

Reply:
Please do